site/_components/item-grid.njk

```diff
@@ -24,9 +25,9 @@
   <p class="type--large item-grid__subtitle">{{modifiers.subtitle}}</p>
   {% endif %}
   <div class="item-grid__content">{% for item in items %} {% if modifiers.variant == 'icon' %} {{iconCard(item, defaultText)}} {% elif modifiers.variant == 'subnav' %} {% if item.icon and item.desc %} {{cardSubnav(item, locale)}} {% endif %} {% elif modifiers.variant == 'community' %} {{externalCommunity(item.logo, item.topic, item.cta)}} {% else %} {{ card(item) }} {% endif %} {% endfor %}</div>
-  {% if modifiers.cta %}
+  {% if modifiers.cta %} {% set localeCta = { 'url': '/' ~ locale ~ '/' ~ modifiers.cta.url, 'text': modifiers.cta.text }%}
   <footer class="item-grid__footer">
-    <a class="cta cta--medium" href="{{modifiers.cta.url}}">{{modifiers.cta.text}}</a>
+    <a class="cta cta--medium" href="{{localeCta.url}}">{{localeCta.text}}</a>
```

Comment:
```suggestion
    <a class="cta cta--medium" href="{% if locale is defined %}/{{locale}}/{% endif %}{{modifiers.cta.url}}">{{modifiers.cta.text}}</a>
```

Why add the complexity of creating a new variable when we can just do it directly here? This also adds a check to see if locale is defined and only prefixes if it is in order to preserve existing functionality